source/views/components/html-view.js

```diff
@@ -8,7 +8,10 @@ export class HtmlView extends React.Component {
   props: {
     html: string,
     baseUrl?: ?string,
-    style?: ?string,
+    style?: ?any,
```

Comment:
This shouldn't be any – it should be `number | Object | Array<number | Object>`… which I really ought to make into a, like, StylePropType or something, so we don't have to keep copying this around.